frb_codegen/src/parser/ty.rs

```diff
@@ -250,6 +289,12 @@ impl<'a> TypeParser<'a> {
                 .or_else(|| {
                     if ident_string == "String" {
                         Some(IrType::Delegate(IrTypeDelegate::String))
+                    } else if ident_string == "Error"
+                        && p.path_segments.iter().any(|x| x.ident == "anyhow")
```

Comment:
uber-nit: Btw I see a constant `ANYHOW_IDENT`. Shall we use that here, or even extract more literals to constants